src/block.json

```
@@ -32,6 +32,22 @@
 		"exclusion": {
 			"type": "boolean",
 			"default": false
+		},
+		"displayPeriodSetting": {
+			"type": "string",
+			"default": "none"
+		},
+		"periodSpecificationMethod": {
+			"type": "string",
+			"default": "direct"
+		},
+		"displayPeriodValue": {
+			"type": "string",
+			"default": ""
+		},
+		"referCustomFieldValue": {
```

```suggestion
		"referCustomFieldName": {
```
@osmdik ここ、入力してもらうのはカスタムフィールド名なので、Name の方がよくない？どうかな？